diablo/jobs/instructor_emails_job.py

```diff
@@ -96,10 +96,19 @@ def _get_interpolate_content(template):
                                 We are unable to notify {course['label']} instructors of room change.
                             """)
                 else:
+                    kaltura_url = f'{app.config["KALTURA_MEDIA_SPACE_URL"]}/recscheduling/index/edit-event/eventid/{scheduled.kalturaScheduleId}'
```

Comment:
@johncrossman Just checking, `recscheduling` is not a typo?

Reply:
Not a typo. Kaltura must answer for that strange spelling. Just the same, good catch.